bin/post_compile

```diff
@@ -4,15 +4,21 @@
 # Make non-zero exit codes & other errors fatal.
 set -euo pipefail
 
-# Create a `prod/` directory containing built/minified versions of the frontend assets.
+# Create a `build/` directory containing built/minified versions of the frontend assets.
 # Uses the node binaries/packages installed by the nodejs buildpack previously.
 npm install && npm run build
 
+# Pull in the Pyodide static resources from a Github release
+export PYODIDE_VERSION=0.1.10
+wget https://github.com/iodide-project/pyodide/releases/download/$PYODIDE_VERSION/pyodide-build-$PYODIDE_VERSION.tar.bz2
+mkdir build/pyodide
+tar -jxf pyodide-build-$PYODIDE_VERSION.tar.bz2 -C build/pyodide
+
 # Generate gzipped versions of files that would benefit from compression, that
 # WhiteNoise can then serve in preference to the originals. This is required
 # since WhiteNoise's Django storage backend only gzips assets handled by
 # collectstatic, and so does not affect files in the `dist/` directory.
-python -m whitenoise.compress prod
+python -m whitenoise.compress build
```

Comment:
I am curious, have we verified that this works recently? :) I imagine compressing the pyodide files correctly/incorrectly would have a large impact on performance. I can take a look at this if you like.

Reply:
I suspect this hasn't been working since the change from `prod` to `build` as output of `npm run build`.  Should help with Iodide itself.

The Pyodide data files are LZ4 compressed so that they can be partially block-decompressed in the client.  Gzip should probably be able to compress them further, but not a ton.  None of the .js files are compressed, though, just minimized.  We should confirm all of this works and helps.  If not, maybe there's a way to tell whitenoise to just compress `build/*` but not `build/pyodide/*`.

Reply:
Looks like the potential space savings are quite considerable with gzip, even moreso with brotli compression (supported by basically all major browsers at this point):

```
(venv)Hammersmith:pyodide wlach$ python -m whitenoise.compress .
Brotli compressed ./py.data (336K -> 200K)
Gzip compressed ./py.data (336K -> 226K)
Brotli compressed ./kiwisolver.js (6K -> 2K)
Gzip compressed ./kiwisolver.js (6K -> 2K)
Brotli compressed ./setuptools.data (1742K -> 790K)
Gzip compressed ./setuptools.data (1742K -> 1205K)
Brotli compressed ./python.html (6K -> 2K)
Gzip compressed ./python.html (6K -> 2K)
Brotli compressed ./pluggy.data (51K -> 29K)
Gzip compressed ./pluggy.data (51K -> 31K)
Brotli compressed ./pyodide.asm.js (3035K -> 283K)
Gzip compressed ./pyodide.asm.js (3035K -> 372K)
Brotli compressed ./atomicwrites.data (15K -> 7K)
Gzip compressed ./atomicwrites.data (15K -> 8K)
Brotli compressed ./pytz.data (653K -> 215K)
Gzip compressed ./pytz.data (653K -> 330K)
Brotli compressed ./pyparsing.data (268K -> 151K)
Gzip compressed ./pyparsing.data (268K -> 176K)
Brotli compressed ./pandas.data (21820K -> 10135K)
Gzip compressed ./pandas.data (21820K -> 13061K)
Brotli compressed ./test.data (8067K -> 4424K)
Gzip compressed ./test.data (8067K -> 5223K)
Brotli compressed ./test.js (229K -> 54K)
Gzip compressed ./test.js (229K -> 64K)
Brotli compressed ./numpy.js (206K -> 50K)
Gzip compressed ./numpy.js (206K -> 59K)
Brotli compressed ./matplotlib.html (6K -> 2K)
Gzip compressed ./matplotlib.html (6K -> 2K)
Brotli compressed ./more-itertools.data (239K -> 134K)
Gzip compressed ./more-itertools.data (239K -> 156K)
Brotli compressed ./nose.data (585K -> 328K)
Gzip compressed ./nose.data (585K -> 390K)
Brotli compressed ./numpy.data (9671K -> 4866K)
Gzip compressed ./numpy.data (9671K -> 6248K)
Brotli compressed ./more-itertools.js (9K -> 2K)
Gzip compressed ./more-itertools.js (9K -> 3K)
Brotli compressed ./setuptools.js (46K -> 9K)
Gzip compressed ./setuptools.js (46K -> 12K)
Brotli compressed ./pyodide.asm.wasm (10018K -> 1978K)
Gzip compressed ./pyodide.asm.wasm (10018K -> 2747K)
Brotli compressed ./pandas.js (509K -> 123K)
Gzip compressed ./pandas.js (509K -> 141K)
Brotli compressed ./pluggy.js (7K -> 1K)
Gzip compressed ./pluggy.js (7K -> 2K)
Brotli compressed ./atomicwrites.js (5K -> 1K)
Gzip compressed ./atomicwrites.js (5K -> 1K)
Brotli compressed ./attrs.data (104K -> 59K)
Gzip compressed ./attrs.data (104K -> 68K)
Brotli compressed ./matplotlib.js (253K -> 48K)
Gzip compressed ./matplotlib.js (253K -> 73K)
Brotli compressed ./kiwisolver.data (77K -> 43K)
Gzip compressed ./kiwisolver.data (77K -> 48K)
Brotli compressed ./matplotlib-sideload.html (2K -> 0K)
Gzip compressed ./matplotlib-sideload.html (2K -> 0K)
Brotli compressed ./renderedhtml.css (3K -> 0K)
Gzip compressed ./renderedhtml.css (3K -> 0K)
Brotli compressed ./attrs.js (8K -> 2K)
Gzip compressed ./attrs.js (8K -> 2K)
Brotli compressed ./nose.js (22K -> 4K)
Gzip compressed ./nose.js (22K -> 6K)
Brotli compressed ./test.html (0K -> 0K)
Gzip compressed ./test.html (0K -> 0K)
Brotli compressed ./joblib.data (480K -> 272K)
Gzip compressed ./joblib.data (480K -> 317K)
Brotli compressed ./pyodide_dev.js (11K -> 2K)
Gzip compressed ./pyodide_dev.js (11K -> 3K)
Brotli compressed ./pytest.data (791K -> 459K)
Gzip compressed ./pytest.data (791K -> 531K)
Brotli compressed ./pyparsing.js (8K -> 2K)
Gzip compressed ./pyparsing.js (8K -> 3K)
Brotli compressed ./pyodide.js (11K -> 2K)
Gzip compressed ./pyodide.js (11K -> 3K)
Brotli compressed ./packages.json (1K -> 0K)
Gzip compressed ./packages.json (1K -> 0K)
Brotli compressed ./py.js (17K -> 3K)
Gzip compressed ./py.js (17K -> 4K)
Brotli compressed ./python-dateutil.js (14K -> 3K)
Gzip compressed ./python-dateutil.js (14K -> 4K)
Brotli compressed ./pytest.js (25K -> 5K)
Gzip compressed ./pytest.js (25K -> 7K)
Brotli compressed ./pytz.js (75K -> 10K)
Gzip compressed ./pytz.js (75K -> 13K)
Brotli compressed ./pyodide.asm.data.js (136K -> 35K)
Gzip compressed ./pyodide.asm.data.js (136K -> 42K)
Brotli compressed ./python_dev.html (1K -> 0K)
Gzip compressed ./python_dev.html (1K -> 0K)
Brotli compressed ./pyodide.asm.data (6579K -> 3631K)
Gzip compressed ./pyodide.asm.data (6579K -> 4446K)
Brotli compressed ./python-dateutil.data (409K -> 286K)
Gzip compressed ./python-dateutil.data (409K -> 317K)
Brotli compressed ./cycler.js (5K -> 1K)
Gzip compressed ./cycler.js (5K -> 2K)
Brotli compressed ./cycler.data (24K -> 12K)
Gzip compressed ./cycler.data (24K -> 13K)
Brotli compressed ./matplotlib.data (15500K -> 8808K)
Gzip compressed ./matplotlib.data (15500K -> 11356K)
Brotli compressed ./joblib.js (27K -> 4K)
Gzip compressed ./joblib.js (27K -> 6K)
```

Reply:
> Brotli compressed ./numpy.data (9671K -> 4866K)
Gzip compressed ./numpy.data (9671K -> 6248K)

Wow, Brotli compression seems to work really well!